source/assets/styles/pages/index/_scroll_link.scss

```diff
@@ -1,6 +1,8 @@
 .scroll-link {
   @include heading;
   display: block;
+  font-family: $font-family-body;
```

Comment:
It's the `@include heading;` above that changed the `.scroll_link`s font-family, so we'd probably start by getting rid of that, if we want regular body font.